pkg/client/build_test.go

```diff
@@ -531,14 +532,15 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 					h.AssertNil(t, fakeRunImage.SetLabel("io.buildpacks.stack.id", "other.stack"))
 				})
 
-				it("errors", func() {
-					h.AssertError(t, subject.Build(context.TODO(), BuildOptions{
+				it("warning", func() {
+					subject.version = "0.37.0"
```

Comment:
@joeybrown do you need this one? I think we didn't do anything related to the version in `build.go`